Formula/vlmcsd.rb

```diff
@@ -0,0 +1,28 @@
+class Vlmcsd < Formula
+  desc "KMS Emulator in C"
+  homepage "https://github.com/Wind4/vlmcsd"
+  url "https://github.com/Wind4/vlmcsd/archive/svn1113.tar.gz"
+  version "svn1113"
+  sha256 "62f55c48f5de1249c2348ab6b96dabbe7e38899230954b0c8774efb01d9c42cc"
+  head "https://github.com/Wind4/vlmcsd.git"
+
+  depends_on "gcc" => :build
```

Comment:
Doesn't it work with LLVM?

Reply:
It supports clang, but I have not tested

Reply:
I tried to compile with clang and it works